src/Tribe/Views/V2/Views/Month_View.php

```diff
@@ -351,7 +351,7 @@ static function ( $event ) use ( $date_object ) {
 
 			$day_data = [
 				'date'             => $day_date,
-				'is_start_of_week' => $start_of_week === $date_object->format( 'w' ),
+				'is_start_of_week' => (int) $start_of_week === (int) $date_object->format( 'w' ),
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: A cast statement must not be followed by a space
* :no_entry_sign: A cast statement must not be followed by a space
* :warning: Line exceeds 80 characters; contains 97 characters

<sub>hash: cf277fe44bf1bc7d13fb7a3cca90b6e0</sub>